src/tests/ftest/scripts/setup_nodes.sh

```diff
@@ -72,8 +72,12 @@ fi
 # make sure to set up for daos_agent. The test harness will take care of
 # creating the /var/run/daos_{agent,server} directories when needed.
 sudo bash -c "set -ex
+if [ -f /tmp/test.cov ]; then
+    rm /tmp/test.cov
+fi
```

Comment:
```suggestion
rm -f /tmp/test.cov
```

Is just as effective as this without the extra overhead of doing a test first.